### PR TITLE
careful filtering of library kind

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem @ahsu1230 

I want to be cautious about the new library kind. We can always let them pass the filter in the future.

### CODE REVIEW

**Comment by ahsu1230** [comment]
:+1:

File: kifi-backend/modules/cortex/app/com/keepit/cortex/models/lda/LDALibraryUpdater.scala [hunk 1]
```diff
@@ -35,6 +35,8 @@ class LDALibraryUpdaterImpl @Inject() (
 
   type Auxiliary = (Option[LDATopic], Option[LDATopic], Option[LDATopic], Option[Float], Option[Float]) // (1st topic, 2nd topic, 3rd topic, 1st topic score, entropy)
 
+  private def acceptLibraryKind(x: LibraryKind): Boolean = (x == LibraryKind.USER_CREATED)
```
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/models/lda/LDALibraryUpdater.scala:38]: Oh, we should definitely accept the system_persona libraries. They should as much as possible be treated like user_created.

**Comment by yingjieMiao** [comment]
I would like to be cautious at this moment. 

Once these things start to flood in, they will have impact on cortex services. For example, the library-library related graph will grow significantly because of these semi-auto-generated libraries. 

And people might start complaining 'their persona libraries' get recommended (I remember the debate on wether those libraries should be public or private) 

Tightening the filter right now doesn't make it difficult later to include these libraries. However, once these libraries get into library_lda_table, that table doesn't track libary_kind (cortex_library table does that). It's going to be harder to filter these things out.

I will keep the filter tight for now, until products people has thought through it. (After that, I may need to do multiple changes on cortex side)

**Comment by stkem** [comment]
Ok, but we musn't forget about this. We will need to be able to recommend these persona libraries sooner or later and we definitely want to see related libraries _on_ the persona libraries.

**Pull request merged**